Demo/src/main/java/com/paypal/android/DemoApplication.kt

```diff
@@ -0,0 +1,26 @@
+package com.paypal.android
+
+import android.app.Application
+import com.paypal.android.di.apiModule
+import com.paypal.android.di.networkModule
+import org.koin.android.ext.koin.androidContext
+import org.koin.core.context.startKoin
+
+class DemoApplication : Application() {
+
+    override fun onCreate() {
+        super.onCreate()
+
+        startKoin {
```

Comment:
randomq: is there a reason we chose Koin over Hilt?

Reply:
same question came up when syncing yesterday, it was more like a quick implementation decision, we thought about discussing this with the SDK team to decide whether we would move to Hilt or leave it as it is, what are your thoughts? I could move it to Hilt if you guys think it is the best approach

Reply:
My vote is for Hilt! I've run into runtime issues with Koin which would have been caught with Dagger/Hilt.

Reply:
I personally prefer Jetpack libraries since there's consistency within the Jetpack ecosystem. Interop with other Jetpack libraries for example. I'm not sure how others feel, but I'd vote for Hilt for `di`.